ci-operator/step-registry/ipi/aws/ovn/pre/ipi-aws-ovn-pre-chain.metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+	"path": "ipi/aws/ovn/pre/ipi-aws-ovn-pre-chain.yaml",
+	"owners": {
+		"approvers": [
+			"aravindhp",
+			"ravisantoshgudimetla"
+		],
+		"reviewers": [
+			"aravindhp",
+			"ravisantoshgudimetla"
+		]
+	}
+}
```

Comment:
nit: missing newline

Reply:
Done

Reply:
@LorbusChris adding newline on those metadata JSON files makes the CI failing on ci/prow/step-registry-metadata

New lines at the end of those metadata files are removed by the `make update` jobs